src/components/Games/Games.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGames } from '../../store/actions';
+import GamesItem from './GamesItem';
+import classes from './games.module.scss';
+
+const Games = () => {
+  const [loading, setLoading] = useState(true);
+  const games = useSelector((state) => state.games);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getGames());
+    setLoading(false);
+  }, []);
+
+  return (
+    <div className={classes.card}>
+      {
+        !loading ? (
+          games?.map((game) => (
+            <GamesItem
+              key={game.id}
+              id={game.id}
+              title={game.title}
+              released={game.released}
+              image={game.image}
+              gener={game.gener}
+            />
+          ))
+        ) : (
+          <div className="spinner-border text-primary" role="status">
+            <span className="sr-only">Loading...</span>
+          </div>
+
+        )
+        }
+    </div>
+
```

Comment:
- [ ] Hi @AACHOURMOHAMED, based on the requirements of this project, the home page has a list of categories :+1: that could be filtered using some parameters e.g. game title. To achieve this, you'll need to use a `Filter stateless` component to filter data retrieved from the API. Kindly fix this issue and move a step towards the approval of this project.

Reply:
hey @FrederickMih 
Should I include for example the most popular games
Thank you

Reply:
@AACHOURMOHAMED. You'll require a search field on the top of your Home page such that when you introduce a title(example Portar 2) of a game, it'll retrieve all games with titles beginning with `P`. The search field should be something like the screenshot below:

![Screenshot from 2022-08-31 19-01-39](https://user-images.githubusercontent.com/63087491/187748116-1e3ed2fa-c50a-4704-a2ca-b17a59d3673f.png)


The filtering should be done in a Filter stateless component